package.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "@truex/ctv-web-reference-app",
+    "version": "1.0.0",
+    "description": "Sample app to demonstrate how to integrate true[X]'s CTV Web ad renderer",
+    "main": "index.js",
+    "public": true,
+    "scripts": {
+        "clean": "rm -rf ./dist/*",
+        "start": "webpack-serve --host 0.0.0.0 --port 8080",
+        "build": "npm run clean && webpack",
+        "build-ps4": "run-func ./platforms/PS4/builder buildLauncher"
+    },
+    "devDependencies": {
+        "@babel/code-frame": "^7.8.3",
+        "@babel/core": "^7.8.3",
+        "@babel/plugin-syntax-dynamic-import": "^7.7.4",
+        "@babel/plugin-transform-async-to-generator": "^7.2.0",
+        "@babel/plugin-transform-runtime": "^7.2.0",
+        "@babel/preset-env": "^7.4.1",
+        "@babel/runtime": "^7.4.1",
+        "babel-loader": "^8.0.6",
+        "css-loader": "^3.5.3",
+        "file-loader": "^6.0.0",
+        "html-loader": "^1.1.0",
+        "style-loader": "^1.2.1",
+        "svg-inline-loader": "^0.8.2",
+        "html-webpack-plugin": "^3.2.0",
+        "copy-webpack-plugin": "^5.1.1",
+        "node-sass": "^4.14.0",
+        "run-func": "^1.0.5",
+        "sass-loader": "^8.0.2",
+        "webpack": "^4.12.0",
+        "webpack-cli": "^3.0.8",
+        "webpack-dev-server": "^3.1.4",
+        "webpack-serve": "^1.0.4"
+    },
+    "dependencies": {
+        "core-js": "^3.6.4",
+        "@truex/ctv-ad-renderer": ">=0.0.0 <=1.0.0",
```

Comment:
the rule is to allow the "automatic" support up to and including 1.0.0, basically to automatically get the latest.

Once TAR is public things have to be done more carefully, and this will be changed to an explicit version scheme like ^1.0.0